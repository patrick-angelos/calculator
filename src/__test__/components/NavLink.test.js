import React from 'react';
import renderer from 'react-test-renderer';
import NavLink from '../../components/NavLink';

describe('NavLink', () => {
  it('renders correctly', () => {
    const tree = renderer
    .create(<NavLink path='/' name='test'/>)
    .toJSON();
    expect(tree).toMatchSnapshot();
  });
});