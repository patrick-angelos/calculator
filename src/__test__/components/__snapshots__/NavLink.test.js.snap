// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLink renders correctly 1`] = `
<a
  className="m-l-10 link"
  href="/"
>
  test
</a>
`;
